liquibase-service/src/main/resources/db/changelog/db.rdb_modern.changelog-16.1.yaml

```diff
@@ -557,4 +557,11 @@ databaseChangeLog:
       changes:
         - sqlFile:
             path: 004-v_getobstxt-001.sql
+            splitStatements: false
+  - changeSet:
+      id: 75
```

Comment:
Conflicting change. The latest id in main branch is 79. Please merge main to this branch.